src/libsyntax/ast.rs

```diff
@@ -178,9 +178,11 @@ pub struct LifetimeDef {
     pub bounds: Vec<Lifetime>
 }
 
-/// A "Path" is essentially Rust's notion of a name; for instance:
-/// std::cmp::PartialEq  .  It's represented as a sequence of identifiers,
+/// A "Path" is essentially Rust's notion of a name.
+/// It's represented as a sequence of identifiers,
 /// along with a bunch of supporting information.
+///
+/// e.g. `std::cmp::PartialEq`
```

Comment:
This seems disconnected from the other text. Maybe you can expand this to a whole sentence and like

> An example for such a path is `std::cmp::PartialEq`, or `std::os::unix::fs::OpenOptionsExt`.